README.md

```diff
@@ -127,6 +127,9 @@ cargo install --locked cargo-pgrx
 
 **Important:** `cargo-pgrx` **must** be built using the same compiler as you'll use to build the rest of your project. Every time you update your Rust toolchain you have to also manually reinstall `cargo-pgrx`. See [Upgrading](#Upgrading), below.
 
+Next, install the dependencies (package names are for Debian-likes):
+- `libreadline-dev`
+
```

Comment:
Can you lift this into the System Requirements, or alternatively move the Getting Started above the System Requirements?

Reply:
oh. hmm. "System Requirements" already links to https://wiki.postgresql.org/wiki/Compile_and_Install_from_source_code and documents that everything there is required, i just missed it when going through the readme. should i just close this PR?

Reply:
@jyn514 I think we should inline the requirements from there and mention that it is an authoritative copy.

Reply:
People are going to have their first-time experience with building Postgres here, probably, and while we can probably put them all on one line, we don't need to spare a few bytes.